### PR TITLE
SSR server and boilerplate

### PR DESCRIPTION
**Pull request by wafflepie**

### CODE REVIEW

**Comment by wafflepie** [comment]
Sorry this PR grew out of hand in size, but I finally managed to get SSR working with Liferay! The boilerplate will be added in another PR (already got it working in my fork).

File: greenkeeper.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
 				"packages/babel-preset-react-union/package.json",
 				"packages/eslint-config-react-union/package.json",
 				"packages/react-union-scripts/package.json",
-				"packages/react-union/package.json"
+				"packages/react-union/package.json",
+				"packages/react-union-ssr-server/package.json"
```
Comment (wafflepie) [inline greenkeeper.json]: Oops, this is missing the new boilerplate, I'll fix that.

File: packages/babel-preset-react-union/src/index.js [hunk 2]
```diff
@@ -53,7 +49,7 @@ module.exports = () => ({
 				regenerator: true,
 			},
 		],
-		cjs && ['@babel/plugin-transform-modules-commonjs', { loose }],
+		'@babel/plugin-transform-modules-commonjs',
```
Comment (wafflepie) [inline packages/babel-preset-react-union/src/index.js]: I'm not exactly sure if this is correct, but it fixed a thing for me and did not break anything. I'll try and investigate.

**Comment by tommmyy** [comment]
Holy moly that is a large PR! Thank you for the work!

**Pull request merged**